content/how-programs-work/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Chapter 2: How Programs Work"
+date: 2023-07-13T13:40:55-05:00
+draft: false
+weight: 2
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Major Concepts 
+
```

Comment:
I'm not sure whether this serves as the content page, but there seems to be a couple problems if it were: 1) under Headings Major Concepts and Key Terminology, there isn't any other content; 2) the subheadings are not consistent with the the structure of each sub-chapter. For example, "Introduction" has "Program" and "Algorithm" here, but in the "Introduction" sub-chapter, there is only "Algorithms" section but not "Program" section.

Reply:
Fixed both of the above issues.